### PR TITLE
Improvement: Add count as template modifier so you can check the amount of items in the template

### PR DESCRIPTION
**Pull request by carakas**

### CODE REVIEW

File: src/Backend/Core/Engine/TemplateModifiers.php [hunk 1]
```diff
@@ -307,4 +307,16 @@ public static function showBool($status, $reverse = false)
...
+    /**
+     * Returns the count of the count of the array.
+     *
+     * @param array $data
+     *
+     * @return int
+     */
+    public static function count(array $data)
+    {
+        return count($data);
```
Comment (WouterSioen) [inline src/Backend/Core/Engine/TemplateModifiers.php]: Isn't there a Common TemplateModifiers class?

Comment (carakas) [inline src/Backend/Core/Engine/TemplateModifiers.php]: No there isn't, but maybe there should be one

Comment (WouterSioen) [inline src/Backend/Core/Engine/TemplateModifiers.php]: this could be a different pull request :)

Comment (carakas) [inline src/Backend/Core/Engine/TemplateModifiers.php]: you have a point there, I will look at this tonight

Comment (WouterSioen) [inline src/Backend/Core/Engine/TemplateModifiers.php]: I would wait for the twig integration ;) modifiers are completely reworked in there ;)

Comment (carakas) [inline src/Backend/Core/Engine/TemplateModifiers.php]: oke, let me know when you are finished with that :D
